packages/plugins/src/cli/deploy.ts

```diff
@@ -214,51 +196,7 @@ export const deploy = async (
 
   checkLibraryVersion(deploymentInfo.sphinxLibraryVersion)
 
-  const dryRunPath = getFoundrySingleChainDryRunPath(
-    broadcastFolder,
-    scriptPath,
-    chainId.toString(),
-    `sphinxCollectDeployment`
-  )
-  const dryRunFile = readFoundrySingleChainDryRun(
-    broadcastFolder,
-    scriptPath,
-    deploymentInfo.chainId,
-    `sphinxCollectDeployment`,
-    dateBeforeForgeScriptCollect
-  )
-
-  // Check if the dry run file exists. If it doesn't, this must mean that the deployment is empty.
-  // We return early in this case.
-  if (!dryRunFile) {
-    spinner.info(`Nothing to deploy. Exiting early.`)
-    return {}
```

Comment:
Moved this later in the function because we need the `actionInputs` to determine whether the deployment is empty.